### PR TITLE
master-qa-16320

### PR DESCRIPTION
**Pull request by brianwulbern**
https://issues.liferay.com/browse/LRQA-16320

### CODE REVIEW

**Comment by brianwulbern** [comment]
@evannagayama 
While this commit doesn't look like anything was changed, I had to change the encoding on the dash to make sure it remains...a dash.  First, I saved the following line to the top of the macro:
```
<?xml version="1.0" encoding="UTF-8"?> 
```
Saved and closed the macro, re-opened and removed the "-", saved and closed again, re-opened and added "-", saved and closed, re-opened and removed the UTF-8 encoding line from above.  

This was all by Greg Amerson's suggestion, telling us that somehow, because the regex character was getting parsed differently in Sublime, Eclipse, and Chrome, that we needed to declare the encoding language and set it, so we can save the intended "-" once, and for all.  So this explains why it looks like nothing is getting changed.

tl;dr, I updated the "-" to be the UTF-8 dash, instead of whatever it was before.

**Comment by evannagayama** [comment]
Cool, thanks for fixing that BWul

**Comment by evannagayama** [comment]
Pushed to BChiu.
https://github.com/brianchiu/liferay-portal/pull/435
